XAlignConfigFrame.java

```diff
@@ -0,0 +1,341 @@
+/*

```

Comment:
It seems that this file is already under `unitex/src/fr/umlv/unitex/frames/XAlignConfigFrame.java` is there any reason to try to create a new copy under the root?